sandbox/cgroup/export_test.go

```diff
@@ -111,3 +111,7 @@ func MockCgroupsFilePath(path string) (restore func()) {
 	cgroupsFilePath = path
 	return r
 }
+
+func MockMonitorFullDelete(name string, folders []string, channel chan string) error {
```

Comment:
Naming here is off, because you are not mocking anything, you are wrapping the call.
```suggestion
func MonitorFullDelete(name string, folders []string, channel chan string) error {
```

Reply:
Done.